lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -191,8 +191,10 @@ class ConvertAtenMatmulOp : public OpConversionPattern<AtenMatmulOp> {
     Value lhs = adaptor.getSelf();
     Value rhs = adaptor.getOther();
 
-    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter))) {
+      llvm::errs() << "NOT VALID\n";
       return failure();
```

Comment:
is the `llvm::errs()` here a remnant from debugging? If not, I think we should use `notifyMatchFailure` to be consistent with other patterns.

Reply:
Woops, slipped through. Deleted